From e0fad2cb0d2083028effc91061a739d3c0901495 Mon Sep 17 00:00:00 2001
From: =?UTF-8?q?S=C3=B6ren=20Tempel?= <soeren+git@soeren-tempel.net>
Date: Sat, 1 Mar 2025 22:46:49 +0100
Subject: [PATCH] abuild: In default -openrc subpackage also move /etc/user
 files

OpenRC 0.60 added experimental support for user services. These ship
their init.d and conf.d files in /etc/user. Hence, we should also move
these subdirectories to the -openrc subpackage.
---
 abuild.in | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/abuild.in b/abuild.in
index 3eec399..9ca0154 100644
--- a/abuild.in
+++ b/abuild.in
@@ -2140,7 +2140,7 @@ default_openrc() {
 	pkgdesc="$pkgdesc (OpenRC init scripts)"
 	install_if="openrc ${subpkgname%-openrc}=$pkgver-r$pkgrel"
 
-	amove etc/conf.d etc/init.d || :
+	amove etc/conf.d etc/init.d etc/user/conf.d etc/user/init.d || :
 }
 
 openrc() {
